tfhe/src/shortint/server_key/mod.rs

```diff
@@ -363,6 +363,41 @@ impl ServerKey {
         })
     }
 
+    /// Given a function as input, constructs the lookup table working on the message bits
+    /// Carry bits are ignored
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::shortint::gen_keys;
+    /// use tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS;
+    ///
+    /// // Generate the client key and the server key:
+    /// let (cks, sks) = gen_keys(PARAM_MESSAGE_2_CARRY_2_KS_PBS);
+    ///
+    /// let msg = 3;
+    ///
+    /// let ct = cks.encrypt(msg);
+    ///
+    /// // Generate the lookup table for the function f: x -> x^2 mod 2^2
+    /// let f = |x| x ^ 2 % 4;
```

Comment:
use the msg mod from parameters directly ?

Reply:
Done